test/functional/test_naming_scheme.py

```diff
@@ -1,22 +1,12 @@
-import glob
-import os
-
 import pytest
 
-from taskotron_python_versions.common import Package
 from taskotron_python_versions.naming_scheme import (
     has_pythonX_package,
     is_unversioned,
     check_naming_policy,
 )
 
-
-def pkg(filename):
-    return os.path.join(os.path.dirname(__file__), '..', 'fixtures', filename)
-
-
-def gpkg(pkgglob):
-    return Package(glob.glob(pkg(pkgglob))[0])
+from .test_common import gpkg
```

Comment:
I would not use the name prefixed with `test_` for common functions. Maybe just `common`?